templates/controller-mutatingwebhookconfiguration.yaml

```
@@ -46,32 +66,34 @@ webhooks:
         - UPDATE
       resources:
         - serviceresolvers
+  sideEffects: None
 - clientConfig:
-    caBundle: {{ quote }}
+    caBundle: Cg==
     service:
       name: {{ template "consul.fullname" . }}-controller-webhook
       namespace: {{ .Release.Namespace }}
-      path: /mutate-v1alpha1-proxydefaults
+      path: /mutate-v1alpha1-servicerouter
   failurePolicy: Fail
-  name: mutate-proxydefaults.consul.hashicorp.com
+  name: mutate-servicerouter.consul.hashicorp.com
   rules:
-  - apiGroups:
-    - consul.hashicorp.com
-    apiVersions:
-    - v1alpha1
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - proxydefaults
+    - apiGroups:
+        - consul.hashicorp.com
+      apiVersions:
+        - v1alpha1
+      operations:
+        - CREATE
+        - UPDATE
+      resources:
+        - servicerouter
```

we seem to be pluralizing other resources 😕 
```suggestion
        - servicerouters
```

---

I think this is coming from the auto-generated code (from kube-builder tags). Let me update it there as well.

---

🙇